modules/metastorage-server/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -179,25 +179,30 @@ public void start() {
 
         ColumnFamilyOptions indexFamilyOptions = new ColumnFamilyOptions(indexOptions);
 
-        List<ColumnFamilyDescriptor> descriptors = Arrays.asList(
+        return List.of(
                 new ColumnFamilyDescriptor(DATA.nameAsBytes(), dataFamilyOptions),
```

Comment:
I wonder if column family options should be closed too

Reply:
I had a thought about that. I think that it is ok if options will be closed by the GC for the following reasons:
1. It is quite cumbersome to pass these options everywhere in order for them to be closed.
2. We create only a couple of these objects during the entire life of an Ignite node, so it's ok if they hang out in the heap for a while.
3. We actually very rarely get rid of these objects.

Reply:
Ok, true